homeassistant/components/xiaomi_miio/config_flow.py

```diff
@@ -46,6 +46,8 @@ def __init__(self):
 
     async def async_step_import(self, conf: dict):
         """Import a configuration from config.yaml."""
+        host = conf.get(CONF_HOST, "no host")
```

Comment:
The host is required according to the platform schema, thus this is always there?
```suggestion
        host = conf.get[CONF_HOST]
```

Reply:
yes you are right, I changed it, thanks